R/annotation-custom.r

```diff
@@ -71,7 +71,10 @@ GeomCustomAnn <- ggproto("GeomCustomAnn", Geom,
   draw_panel = function(data, panel_params, coord, grob, xmin, xmax,
                         ymin, ymax) {
     if (!inherits(coord, "CoordCartesian")) {
-      abort("annotation_custom only works with Cartesian coordinates")
+      cli::cli_abort(c(
+              "{.fn annotation_custom} only works with Cartesian coordinates",
+        "i" = "Only use {.fn coord_cartesian} to modify the coordinate system"
+      ))
```

Comment:
```suggestion
      cli::cli_abort(c(
        "{.fn annotation_custom} only works with Cartesian coordinates",
        "i" = "Only use {.fn coord_cartesian} to modify the coordinate system"
      ))
```

I'm not sure I understand what the `i` bullet means

Reply:
it's meant to say that if you want to change aspects of the coordinate system you must do it through a `coord_cartesian()` call and not with `coord_whatever()`...

I adopted it from the warnings from `coord_sf()` where the user is more likely to add e.g. `coord_cartesian(expand = FALSE` or something like that, but I agree it makes less sense here... should I just remove it